packages/connect/src/protocol-grpc/request-header.ts

```diff
@@ -32,18 +32,16 @@ export function requestHeader(
   timeoutMs: number | undefined,
   userProvidedHeaders: HeadersInit | undefined,
 ): Headers {
-  const result = new Headers(userProvidedHeaders ?? {});
+  const result = new Headers(userProvidedHeaders);
   result.set(
     headerContentType,
     useBinaryFormat ? contentTypeProto : contentTypeJson,
   );
-  // Note that we do not strictly comply with gRPC user agents.
-  // We use "connect-es/1.2.3" where gRPC would use "grpc-es/1.2.3".
-  // See https://github.com/grpc/grpc/blob/c462bb8d485fc1434ecfae438823ca8d14cf3154/doc/PROTOCOL-HTTP2.md#user-agents
```

Comment:
If you follow the link in the comment, you'll see that the gRPC spec recommends that clients provide a user-agent.

Reply:
Yes, and I want to pass my user-agent: to understand who my consumers are, add the runtime version, the library version

Reply:
@timostamm I've updated PR